src/const/faqs.jsx

```diff
@@ -148,9 +148,11 @@ const faqs = [
       </Translation>
     ),
     answer: (
-      <Translation>
-        {(t) => `${t('faq.answer.linkAPI')}`}
-      </Translation>
+      <a href={`${process.env.REACT_APP_API_BASE_URL}/schema/redoc/`}>
```

Comment:
1. https://ipa.dataocean.us/schema/redoc/
2. https://ipa.dataocean.us//schema/redoc/

Reply:
In the .env file REACT_APP_API_BASE_URL = https://ipa.dataocean.us. 
 <a href={`${process.env.REACT_APP_API_BASE_URL}/schema/redoc/`}>